src/argustvrpc.cpp

```diff
@@ -204,7 +204,7 @@ namespace ArgusTV
   std::string GetChannelLogo(const std::string& channelGUID)
   {
 #if defined(TARGET_WINDOWS)
-#if defined(TARGET_WINDOWS_STORE)
+#if defined(UNICODE)
     wchar_t wpath[MAX_PATH];
     GetTempPath(MAX_PATH, wpath);
```

Comment:
Any reason why you not use this [GetTempPathA](https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettemppatha)?

Reply:
See https://github.com/kodi-pvr/pvr.argustv/pull/97/commits/9ace6c1854bea91cc45167fef7a083534ef0e4b2